test/integration/integration_test.py

```diff
@@ -10,24 +10,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from contextlib import contextmanager
-from marquez_client.marquez import MarquezClient
-from marquez_codegen_client.rest import ApiException
-from urllib3.exceptions import (MaxRetryError,
-                                NewConnectionError,
-                                ReadTimeoutError)
-from urllib3.util.retry import Retry
+import subprocess
+import sys
 
 import pytest
 import requests
-import subprocess
-import sys
+from marquez.client import MarquezClient
+from marquez.constants import NOT_FOUND
+from requests import ReadTimeout
+from urllib3.exceptions import MaxRetryError
+from urllib3.util.retry import Retry
 
 MARQUEZ_HOST = "marquez"
 MARQUEZ_PORT = "5000"
 
 
-@pytest.fixture(scope="class")
+@pytest.fixture
+def broken_network():
```

Comment:
is this testing against a real marquez instance? the iptables trick only works if marquez is running somewhere

Reply:
This is Rodrigo's integration test. It's running against a locally-running Marquez instance, set up with docker-compose.

We were having cleanup issues in cases of test failure, so I adjusted `broken_network` to be a test fixture (which is the way to handle these situations).